CitiesSkylinesMods/TMPE
# partial migration of VehicleAI.StartPathFind into harmony.

partially f i x es #895

I have written harmony patches for all `VehicleAI.StartPathFind()` except for `TransportLineAI` (which is complicated)

How to review code:
- decompile CS code and TrafficManager.DLL using the same decompiler.
- take a diff between StartPathFind and TMPE CustomStartPathFind
- make sure all differences are addressed in my harmony patches.
- start the review by looking at `PathManager.CreatePathPatch()`
- then look at each individual path for each subclass of VehicleAI
- see comments in #895.

Patch summary:
- some redirections of `CustomStartPathFind()` call to `OnStartPathFind()`. This is reflected in my patches
- `skipQueue=true` (if spawned)
- PostVanAI, CustomTruckAI has a transpiler to be compatible with 81 tiles.

~~TODOs:
Please tell me what do you think about the `//TODO [issue] ` in my code? should I simply delete them or create new issues for them?~~ I created new issues.
